src/sst/elements/ariel/ariel_shmem.h

```diff
@@ -171,6 +173,15 @@ struct ArielCommand {
         struct {
             uint64_t vaddr;
         } flushline;
+        struct {
```

Comment:
@gvoskuilen Do we want ifdef's around these changes? This is different since you need the cuda libs and, for osseus, there aren't any external deps.